Core/HLE/sceKernelThread.cpp

```diff
@@ -3095,6 +3100,13 @@ void __KernelExecuteMipsCallOnCurrentThread(u32 callId, bool reschedAfter)
 	call->savedV1 = currentMIPS->r[MIPS_REG_V1];
 	call->savedId = cur->currentMipscallId;
 	call->reschedAfter = reschedAfter;
+	
+	// Also need to backup other regs which might be used by the called mips function, needed to prevent corrupting the original regs (ie. Dissidia 012 with more than 2 players, Lord of Arcana)
```

Comment:
Well, that may mean that the callback called something else, or generated another callback.  Right now, callbacks inside other callbacks don't work.

Also, you probably need/want "real clock sync" during netplay.

-[Unknown]